fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/Coalesce.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import java.util.List;
+
+/**
+ * Represents the SQL COALESCE function.
+ */
+public class Coalesce extends VarArgFunction {
+    private static final String FUNCTION_NAME = "COALESCE";
```

Comment:
if this  is private, maybe just return direction  instead  of the constant.

Reply:
dead class